packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -143,7 +144,6 @@ class BaseModelSqlv2 {
           is_group: true,
           logical_op: 'and',
         }),
-        ...(args.filterArr || []),
```

Comment:
what is the reason to remove this line?

Reply:
@wingkwong - which is not required since it's a repetition, copy of ```new Filter({
          children: args.filterArr || [],
          is_group: true,
          logical_op: 'and',
        })```